modules/Media.js

```diff
@@ -31,7 +30,7 @@ class Media extends React.Component {
 
   componentWillMount() {
     let { query } = this.props;
-    const { targetWindow } = this.props;
+    const targetWindow = this.props.targetWindow || window;
 
     if (typeof targetWindow !== "object") return;
```

Comment:
This is the line that checks to make sure we're actually in a browser. It should stay:

```js
if (typeof window !== 'object') return;
```

and then get the `targetWindow` prop afterwards.